docs/content/releases.md

```diff
@@ -6,6 +6,62 @@ doctypes: ["concept"]
+* [3646](https://github.com/nginxinc/kubernetes-ingress/pull/3646) Remove app protect agent.
+* [3507](https://github.com/nginxinc/kubernetes-ingress/pull/3507) Support empty path for ImplementationSpecific pathType.
+* [3482](https://github.com/nginxinc/kubernetes-ingress/pull/3482) Use new NSM Spiffe and Cert rotation library.
+* [3442](https://github.com/nginxinc/kubernetes-ingress/pull/3442) Add websocket protocol option to monitor directive.
+* [3674](https://github.com/nginxinc/kubernetes-ingress/pull/3674) Move NAP DoS chart to new repo
+* [3302](https://github.com/nginxinc/kubernetes-ingress/pull/3302) Make default-server-secret optional.
+* [3586](https://github.com/nginxinc/kubernetes-ingress/pull/3586) Add new labels and metadata to add version information to pods
+
+FIXES:
+* [3463](https://github.com/nginxinc/kubernetes-ingress/pull/3463) Support non-vs created Challenge Ingress.
+* [3475](https://github.com/nginxinc/kubernetes-ingress/pull/3475) Ensure leader election is correctly disabled when option is set to `false` in helm template.
+* [3481](https://github.com/nginxinc/kubernetes-ingress/pull/3481) Add missing OSS internal routes for integration with NSM.
+* [3541](https://github.com/nginxinc/kubernetes-ingress/pull/3541) Ensure non-ready endpoints are not added to upstreams.
+* [3583](https://github.com/nginxinc/kubernetes-ingress/pull/3583) Update keyCache path for JWKs to avoid conflict with OIDC.
+* [3607](https://github.com/nginxinc/kubernetes-ingress/pull/3607) Clear Content-Length headers for requests processed by internal JWKS routes.
+* [3660](https://github.com/nginxinc/kubernetes-ingress/pull/3660) Remove unwanted chars from label value.
+
+HELM CHART:
+* [3581](https://github.com/nginxinc/kubernetes-ingress/pull/3581) Push edge Helm Chart to OCI registries.
+* [3449](https://github.com/nginxinc/kubernetes-ingress/pull/3449) Correct values.schema.json nodeSelector.
+* [3448](https://github.com/nginxinc/kubernetes-ingress/pull/3448) Fix Helm Chart Schema for priorityClassName.
+* [3519](https://github.com/nginxinc/kubernetes-ingress/pull/3519) Add OnDelete to allowed strategy values.
+* [3537](https://github.com/nginxinc/kubernetes-ingress/pull/3537) Update schema references to k8s v1.26.1.
+* [3606](https://github.com/nginxinc/kubernetes-ingress/pull/3606) Fix Helm Chart labels and templates. Move version update to labels.
+
+
+UPGRADE:
+* For NGINX, use the 3.1.0 images from our [DockerHub](https://hub.docker.com/r/nginx/nginx-ingress/tags?page=1&ordering=last_updated&name=3.1.0), [GitHub Container](https://github.com/nginxinc/kubernetes-ingress/pkgs/container/kubernetes-ingress), [Amazon ECR Public Gallery](https://gallery.ecr.aws/nginx/nginx-ingress) or [Quay.io](https://quay.io/repository/nginx/nginx-ingress).
+* For NGINX Plus, use the 3.1.0 images from the F5 Container registry or build your own image using the 3.1.0 source code.
+* For Helm, use version 0.17.0 of the chart.
+
```

Comment:
All major releases should include:

```
SUPPORTED PLATFORMS:

We will provide technical support for NGINX Ingress Controller on any Kubernetes platform that is currently supported by its provider and that passes the Kubernetes conformance tests. This release was fully tested on the following Kubernetes versions: 1.22-1.26.
```
see: https://docs.nginx.com/nginx-ingress-controller/releases/#nginx-ingress-controller-300